### PR TITLE
Color blind themes

### PR DESCRIPTION
**Pull request by alschmiedt**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Proposed Changes
Add color palettes for three types of color blindness: Deuteranopia, Protanopia, and Tritanopia. The Deuteranopia color palette works for Protanopia as well. Also removed modern from the themes in require. Doesn't seem necessary since we still have classic.

Deuteranopia/Protanopia
Screen Shot 2019-10-23 at 12 42 01 PM

Tritanopia
Screen Shot 2019-10-23 at 12 42 15 PM

### Test Coverage

Tested on:

### CODE REVIEW

File: core/requires.js [hunk 1]
```diff
@@ -88,5 +88,6 @@ goog.require('Blockly.geras.Renderer');
 // Classic is the default theme.
 goog.require('Blockly.Themes.Classic');
 goog.require('Blockly.Themes.Dark');
+goog.require('Blockly.Themes.Deuteranopia');
 goog.require('Blockly.Themes.HighContrast');
-goog.require('Blockly.Themes.Modern');
```
Comment (samelhusseini) [inline core/requires.js]: Intentionally removing modern?

Comment (samelhusseini) [inline core/requires.js]: I would comment it out instead, similar to requires that aren't included in the rest of this file.

File: tests/playground.html [hunk 2]
```diff
@@ -584,9 +587,10 @@ <h1>Blockly Playground</h1>
   </form>
   <select id="themeChanger" name="theme" onchange="changeTheme()">
     <option value="classic">Classic</option>
-    <option value="modern">Modern</option>
```
Comment (samelhusseini) [inline tests/playground.html]: Why remove modern from the playground?

Comment (samelhusseini) [reply tests/playground.html]: You can require it separately in the playground.html file

File: core/requires.js (hunk 1 above)
Comment (alschmiedt) [reply core/requires.js]: done

File: tests/playground.html (hunk 2 above)
Comment (alschmiedt) [reply tests/playground.html]: ok sounds good

**Pull request merged**